File: app/src/main/java/com/example/carsapplication/ui/manufacturers/ManufacturerExample.java
package com.example.carsapplication.ui.manufacturers;

public class ManufacturerExample {

    private String title;
    private String country;
    private int imageLogo;

    public ManufacturerExample() {
    }

    public ManufacturerExample(String title, String country, int imageLogo) {
        this.title = title;
        this.country = country;
        this.imageLogo = imageLogo;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public int getImageLogo() {
        return imageLogo;
    }

    public void setImageLogo(int imageLogo) {
        this.imageLogo = imageLogo;
    }
}

File: app/src/main/java/com/example/carsapplication/ui/manufacturers/ManufacturerExample.kt
package com.example.carsapplication.ui.manufacturers

class ManufacturerExample constructor() {
    lateinit var title : String;
    lateinit var country : String;
    var imageLogo : Int = 0;

     constructor (title: String?, country: String?, imageLogo: Int) : this() {
        this.title = title!!
        this.country = country!!
        this.imageLogo = imageLogo
    }
}